lib/l10n/l10n.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class L10n {
+  static final all = [
+    const Locale('en'),
+    const Locale('pl'),
+    const Locale('ru'),
+    const Locale('uk'),
```

Comment:
Ukraine language code is 'ua' I suppose

Reply:
I analyzed this question, it seemed that the Ukrainian language is not supported. 😢